service/kafka-plugin/src/test/java/com/solace/maas/ep/runtime/agent/plugin/kafka/processor/KafkaConsumerGroupProcessorTests.java

```diff
@@ -25,11 +24,11 @@
 import static org.mockito.Mockito.when;
 
 @ActiveProfiles("TEST")
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = TestConfig.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
Thanks for cleaning these up.

Reply:
There wasn't much of a choice considering "TestConfig" can't get imported into these Modules. If the plugin modules need their own TestConfig it will need to be implemented individually.